journal/modernized/02-chapter-1.adoc

```diff
@@ -1,6 +1,8 @@
-== Journal of the Life of William Edmundson.Section I.
+== Journal of the Life of William Edmundson. Section I.
+//Instead of this saying "secton I." do you want it to say Chapter 1? And no Roman numeral? Or are these ones okay?
 
 Giving an account of his birth, parentage, trade, marriage, convincement, etc.
+//^^Does this need to have any subtitle syntax or whatever? It seems to be summarizing the chapter, but I couldn't find anything that fit it in the style guide. Maybe a secondary heading? Idk.
```

Comment:
Yes, I think it would be best to change this to _Chapter_ instead of _Section_.  You can leave the roman numeral for the chapter number. I think that's currently our norm for chapter numbers. 
I'm also thinking we can probably remove the "Journal of the Life of William Edmundson" here, because it seems unnecessary. (although maybe @jaredh159 can confirm both of these decisions.)
So.... looking down through this document a bit, I see that each chapter has something of a chapter synopsis after the chapter heading. So what you're going to want to do is this. 

`== Chapter I`

`[.chapter-synopsis]`
`* An Account of His Birth, Parentage, Trade, Marriage, Convincement, etc.`

Notice I capitalized the main words in the chapter synopsis, and also slightly reworded it. The reason I slightly reworded it is first because later on in the book the synopsis are in the format of lists of topics or events, and not complete sentences, so I thought I would make this one more like the ones that are found throughout the remainder of the book.

Reply:
Yup, both are fine with me, let's remove the "Journal of the Life etc.", and I'm OK switching to chapters.

I actually have an open issue documenting the need to possibly add a concept of an "Intermediate Title Page" for exactly this sort of thing, because I've come up against it a handful of times, and I'm never totally sure what to do, here's the issue link:

https://github.com/friends-library/friends-library/issues/75

But anyway, yeah, I made a note there, so let's just remove it for now, if I later on add the ability to do intermediate titles, I'll restore it at that point.